cis_proc.py

```diff
@@ -94,7 +81,20 @@ def main(argv=None):
 
     with open(args.config, 'r') as fo:
         config_options = json.load(fo)
+    if 'project' not in config_options.keys():
+        raise Exception('Config File must be updated with project field'
+                        'See Sample Config File for More information')
```

Comment:
I prefer newlines surrounding if statements. Can you add one before and after this statement?